### DISCUSSION TITLE
Non-squishy blobs

### DISCUSSION THREAD

**Discussion by Snuggle** [C1710/blobmoji#65] [Style]
I love what you've been doing here with blobmoji, but I'm feeling ![image](https://user-images.githubusercontent.com/26250962/44243618-91b23180-a1bf-11e8-964d-0843e9e2ec70.png) because the sad blob isn't a real squishy blob, but rather a stiff & rigid one.

There are a couple other blobs that have squishy variants which I much much prefer, but they were changed to be more rigid, "standardised" and boring copy with the Android 6.0 -> 7.0 release.

I believe that it's important to preserve the "blobbiness" of the blobmoji, a wonderful and adorable design choice that was slowly stripped away, right before the end of blobmoji in Noto Emoji.

_Sleeping Blob_
![image](https://user-images.githubusercontent.com/26250962/44244140-f8d0e580-a1c1-11e8-850a-4c06793fb790.png) vs ![image](https://user-images.githubusercontent.com/26250962/44244143-fd959980-a1c1-11e8-846e-64632aaa3fb7.png)

_Sad Blobs_
![image](https://user-images.githubusercontent.com/26250962/44244208-506f5100-a1c2-11e8-9179-173fe93479a3.png) vs ![image](https://user-images.githubusercontent.com/26250962/44244216-56fdc880-a1c2-11e8-9731-ec09513736f7.png)

**Notable changes, color-coded - [[Squishy blobs](https://emojipedia.org/google/android-6.0.1/) | [Rigid blobs (Current)](https://emojipedia.org/google/android-7.0/)]**
![image](https://user-images.githubusercontent.com/26250962/44243985-3e40e300-a1c1-11e8-8ad4-56e3f58efa48.png)

In terms of standardization, the only real change is the way that some of these blobs face (Left/right/forward), but most of them are pretty identical.

~

I'd like to hear people's thoughts, too. Do you prefer the squisher blobs? The sad blobs in particular, had a lot of redundant copies of... essentially the same thing.

Blobs are meant to be squishy.

**Comment by Madis0** [comment]
I think the forward-facing emoji are more legible and therefore better in the long term, though the squishy ones could be a sticker pack for some keyboard/messenger (like they are in Allo). 

On a related note, maybe we could blobify the Unicode-defined gender-neutral emoji? E.g the "kiss", "couple with heart" and "family" defaults in Android Pie [have changed.](https://emojipedia.org/google/android-9.0/changed/)

**Comment by C1710** [comment]
Yeah, this is one of the things bothering me the most.  
I'm still undecided on __how__ to make the blobs squishier again, but I can definitely say that this is a goal.

**Comment by Snuggle** [comment]
Most of the faces at the start should be relatively easy to use and you'd be able to just drop them in without issue, but some of the newer things like skin-tone modifiers don't exist for any blobs, so they were replaced with humans in v7.0. Notable examples are the policeblob/policeman and the waving blob.

**Comment by Cubox** [comment]
I personally prefer the older blob emojis (6.0.1 Android) rather than the "new blobs" emojis.
They feel more like individuals and not faces

**Comment by Snuggle** [comment]
Yes, I totally agree! I'm not sure how this could be done, though.
I'd be happy to contribute, but this project doesn't have a contributor's document/guide and I have no clue how this works.

**Comment by C1710** [comment]
@Snuggle To be honest I wouldn't know what to put into a contributor's guide 😅 since this is the first real project I do (at the beginning I didn't even intend to make this fork public, but my local Git server was broken).  

But back to the Blobs: First of all one could simply include the older assets which are also available as SVG (as far as I remember).  
Actually creating _new_ squishy Blobs would be slightly more difficult because of the perspective (which can be applied in Inkscape - although my laptop struggles with this effect).  
Last but not least I'm not 100% sure if this is something good for a font which is supposed to be available for mainstream devices/services (as the Magisk module or Tusky).  
I already tried a different solution - making the front-facing Blobs a bit more squishy, although I haven't found something which can be applied by mass (I'd prefer to do these changes by writing a script for that), doesn't look like the current version and doesn't look scary.

**Comment by C1710** [comment]
Okay. I just made a [CONTRIBUTING.md](https://github.com/C1710/blobmoji/blob/redesign/CONTRIBUTING.md) file on the redesign branch (I'm not sure about it yet and I think it might be good to not do every small change on the master branch)

**Comment by Younes-L** [comment]
I'm not alone ! Long live the true blobs !

**Comment by Madis0** [comment]
[Relevant article worth reading](https://blog.emojipedia.org/googles-three-gender-emoji-future/#yes-but-the-blobs)

**Comment by C1710** [comment]
> People sent a woman in a red dress from iOS or Twitter weren't necessarily pleased at the blob with a rose in its mouth […]

Yeah, and the new genderless version is so much more representing a woman in a red dress 🙄

However I _do_ agree that semantic compatibility is always to be considered.  
My long time goal would be to have some sort of all in one software where you can just choose some preferences for your Blobmoji font and let the scripts generate a suitable font.  
This would give everyone the opportunity to choose their favourite style (which was also the no. 1 reason why I extended the EmojiCompat library to use user-accessible font files for their emoji style).

**Comment by Cubox** [comment]
Is there any work ongoing towards that, or is that just on the todo list?
Having the blobs on my phone is the one thing making me tolerate emojis, so thanks for the work you put into this

**Comment by C1710** [comment]
@Cubox- Well, I wouldn't expect it to be released anytime soon, i.e. probably not in the next weeks or next months. Currently it's not more than a idea; first of all I would even need to understand the whole process that creates these fonts (all the additions I made mostly operate on the source files; I don't really know what happens inside the Makefile...).

So to answer your question: It's on the long time todo list.

**Comment by Secret-chest** [comment]
#148
